tests/testthat/test-methods.R

```diff
@@ -152,16 +145,14 @@ test_that("within.list is dispatched", {
 
 test_that("as_tibble.list is _not_ dispatched directly", {
   skip_if_not_installed("tibble")
-  expect_identical(
-    nrow(tibble::as_tibble(lint(text = "a = 1", linters = assignment_linter()))),
```

Comment:
Dagwood sandwich making it difficult to parse this test. The same for the test below.